raiden-ts/tests/integration/raiden.spec.ts

```diff
@@ -42,7 +42,7 @@ import { RaidenTransfer, RaidenTransferStatus } from 'raiden-ts/transfers/state'
 import { makeSecret, getSecrethash } from 'raiden-ts/transfers/utils';
 import { matrixSetup } from 'raiden-ts/transport/actions';
 import { jsonStringify } from 'raiden-ts/utils/data';
-import { ServiceRegistryFactory } from 'raiden-ts/contracts/ServiceRegistryFactory';
+import type { ServiceRegistryFactory } from 'raiden-ts/contracts';
```

Comment:
I don't think this object exists anymore. Please, clean your environment before building again, this should then be made noticed by `tsc`